### DISCUSSION TITLE
Intel vs AMD processors for GO

### DISCUSSION THREAD

**Discussion by thbeckpr** [GrandOrgue/grandorgue#2043] [General]
My desktop PC has died and I am looking at various types of PCs to replace it with. I am looking at MINI PCs. They are available with Intel or AMD processors. 

Would an AMD Ryzen 7 class processor work well for GO? Or should I stick to Intel. I have heard of problems with the latest generation of Intel processors.

**Comment by davidgritter** [comment]
I have used both AMD and Intel processors, and have concluded that their is hardly any difference in GrandOrgue performance between them.   I presently use a GEEKOM 5x5 inch computer with an eighth generation intel I5 and 32 Gig of RAM, and have had no problems with GrandOrgue.  I also have a dell workstation with a 6 core XEON processor.  I've concluded that having lots of RAM and MORE than one slot for an  NVE SSD is more important than the make or the age of the processor.  64GB RAM should be a starting point, and 128GB allows me to load almost any sample set without limitation. A Limitation of the Geekom processor is that the only way to get good multichannel sound is with an external USB sound card.  I am satisfied with my eight channel ASUS Xonar U7 USB adapter, but many other's I've tried have not worked so well.   A Good starter miniPC is a dell small form factor refurbished computer with 64 GB RAM avalaible for about $250:
![image](https://github.com/user-attachments/assets/0cc1a0a6-07f0-48e5-bb69-e6995f3e36eb)
or a dell workstation (*admittedly not a MINI PC) with 128 GB RAM and space for multiple SSDs for about $400 to $500:
![image](https://github.com/user-attachments/assets/f6af3535-607b-4789-9db3-1213f7a5b913)

**Comment by oleg68** [comment]
Because most of modern sample sets require lots of memory, the memory size is the most important resource for GrandOrgue. I think, when you buy a new PC, 64GB should be a recommended minimum, but the capability of extension up to 128 GB would be a good bonus.

Intel vs AMD do not have much difference for GO. I'd pay attention to the maximum memory size supported by the certain CPU model and I'd choose one that supports more memory.

I'd avoid to use a server platform for GrandOrgue because they usually have aggressive cooling and their fans make a lot of noise. Any fanless model of PC would be preferable.

**Comment by thbeckpr** [comment]
Thanks for the info, guys.

I think I'll go with a lower level mini pc with a ryzen CPU. I was looking at some of the higher end units which have powerful gpus but I certainly don't need that power for GO unless GO can offload computations to the gpu. Oleg, am I correct in thinking that GO doesn't take advantage of gpu processing?

**Reply by oleg68** [reply]
> I think I'll go with a lower level mini pc with a ryzen CPU

Yes, you are right. You do not need a high-end CPU. 4-6 CPU cores (Ryzen 5 and even Ryzen 3) would be enough.

But look at 
- the memory size. Start with 64GB RAM.
- m.2 ssd. 2 TB would be enough for many various sample sets.

> am I correct in thinking that GO doesn't take advantage of gpu processing?

Yes, you are. A simplest GPU even an internal CPU graphics (if existed) would be enough.

**Reply by thbeckpr** [reply]
My current, now defunct, PC only has 24GB and it is just barely sufficient for the sample sets I use. 32GB will be a big help. If I ever acquire a sample set that requires more memory I will upgrade to 64GB at that time. It's hard to find PCs that are configured with 64GB. 

I have found a mini PC with a Ryzen 7 5700U with 32GB expandable to 64GB 1TB SSD expandable to 2TB for $359. Apparently it is very quiet as well. I'm never going to use all the perspectives provided with the new sample sets. I think it will suffice. 

Thanks for your advice.
